static/app/views/alerts/rules/metric/details/header.tsx

```diff
@@ -118,7 +116,7 @@ function DetailsHeader({
             <Access access={['alerts:write']}>
               {({hasAccess}) => (
                 <SnoozeAlert
-                  isSnoozed={isSnoozed}
+                  isSnoozed={rule?.snoozeForEveryone ?? false}
```

Comment:
i miss these `?s`.. coalescing the nulls, optional chaining..... just so much simpler.